Controller tutorial

henrygerardmoore opened the pull request:
### Description

Adds a controller tutorial with the panda arm, closing #38.

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

tylerjw reviewed (approved):
This is awesome, I read through it and it looks really helpful.

tylerjw reviewed (commented):
This seems like it might be more of a HowTo than a tutorial. Do you think it belongs with the other tutorials or should it be somewhere else?

gavanderhoorn commented:
Given the fact that the word *controller* is used for many different things (both in MoveIt as elsewhere), I'd suggest changing this to be clear it's about using a *game controller* or *gamepad* (or something similar).

From the PR title I expected to read about MoveIt controllers, or `ros2_control` controllers or their interaction. The gamepad picture made it clear that's not the case.

Perhaps something like "Using a gamepad for teleoperation of a robot arm" (to make it really clear)?

henrygerardmoore commented:
Changed the title, switched to a how-to, and changed all occurrences of the word "controller" to "gamepad" for clarity. I also changed the prerequisites section to mention how to ensure your controller is working since the rviz tutorial isn't really a prerequisite. Thanks for the feedback all!
